src/core/README.md

```diff
@@ -1,4 +1,5 @@
-# Overview
+# Shared C-core library
 
-This directory contains source code for C library (a.k.a the *gRPC C core*) that provides all gRPC's core functionality through a low level API. Libraries in other languages in this repository (C++, C#, Ruby,
-Python, PHP, NodeJS, Objective-C) are layered on top of this library.
+This library provides all of gRPC's core functionality through a low level API.
+gRPC libraries for the other languages supported in this repo, are built on
+top of this shared C-core library.
```

Comment:
Sorry for being strict. The Core team has tried hard to use just "Core" instead of "C-Core" because it is no longer written only in C. Also, there are other languages supported by gRPC out side of this repo using gRPC Core (see https://github.com/grpc).

Reply:
No worries, that's what reviewers are meant to do!

> The Core team has tried hard to use just "Core" instead of "C-Core" because it is no longer written only in C.

Ok, that makes sense (I'll update that tomorrow).

Note that "C core" is used in the [repo's top-level README](https://github.com/grpc/grpc/blob/master/README.md); should that be updated too?

Reply:
@lidizheng: I've made the update, PTAL.

Reply:
It would be nice to change that as well, I didn't notice before.

Reply:
Ok, I'll submit an update for the top-level README as well.